test/embedded/classes-non-final-method-no-stdlib.swift

```diff
@@ -3,6 +3,6 @@
 // REQUIRES: swift_in_compiler
 
 public class MyClass {
-  func foo<T>(t: T) { } // expected-error {{classes cannot have non-final generic fuctions in embedded Swift}}
+  func foo<T>(t: T) { } // expected-error {{classes cannot have non-final generic functions in embedded Swift}}
```

Comment:
This change is required in #75020 where you fix the actual message template. We should drop it here.

Reply:
reverted